.vscode/settings.json

```
@@ -1,3 +1,6 @@
 {
-    "eslint.workingDirectories": ["./context"]
+    "eslint.workingDirectories": [
+        "./context"
+    ],
+    "python.pythonPath": "/usr/bin/python"
```

```suggestion
    "eslint.workingDirectories": ["./context"]
```

Having IDE settings checked in is a grey-area: Sometimes it makes sense to share settings across a team... other times different people may have different preferences. In this case, different devs will be using python binaries at different locations, so this should be reverted.